fastapi-url-shortener/source_code_final/requirements.txt

```diff
@@ -0,0 +1,16 @@
+anyio==3.6.0
+asgiref==3.5.1
+click==8.1.3
+decorator==5.1.1
+fastapi==0.75.0
+h11==0.13.0
```

Comment:
I seem to have one extra dependency in my virtual environment, but that might be some Linux-specific thing maybe? :thinking: 

```
greenlet==1.1.2
```